CHANGES.rst

```diff
@@ -9,6 +9,8 @@ Change History for ZConfig
 
 - Run tests with pypy and pypy3 as well.
 
+- Host docs at http://zconfig.readthedocs.io
```

Comment:
Ok, some quick experiments:

- https://zconfig.readthedocs.io redirects to https://zconfig.readthedocs.io/en/latest/
- https://zconfig.readthedocs.io/en/latest/ responds over TLS
- https://zconfig.readthedocs.io/en/latest (without trailing slash) redirects to http://zconfig.readthedocs.io.en/latest/ (adding the slash but dropping the https)

Looks like a misconfiguration on RTD.  I'll see if I can find a place to report it.